rpm/cppcheck.spec

```diff
@@ -0,0 +1,51 @@
+Name:           cppcheck
+Version:        2.1
```

Comment:
Question: Do you want that we update this version for every release? Then a comment about that should be added in `createrelease`, we have comments about updating versions there already please adjust it.

Reply:
I’ll automatically put the version in rpm spec file resulting from the build version and I’ll come back with something that doesn't require changing the version manually.

Reply:
I added the code update so that the version updates automatically .